test/source.js

```diff
@@ -32,6 +32,12 @@ describe('Source', () => {
       source.relativeToOffset(2, 50).should.equal(22)
       source.relativeToOffset(1, Infinity).should.equal(1)
     })
+
+    it('returns empty object for out of range params', () => {
```

Comment:
New test covers a previously uncovered return statement for out of range issues in `offsetToOriginalRelative`. Added to bump a rounding error out of the previous build.